pkg/pxc/users/users.go

```diff
@@ -197,3 +197,47 @@ func (u *Manager) UpdateProxyUsers(proxyUsers []SysUser) error {
 
 	return nil
 }
+
+func (u *Manager) UpdateUserGrant(user string) error {
+	tx, err := u.db.Begin()
+	if err != nil {
+		return errors.Wrap(err, "begin transaction")
+	}
+
+	switch user {
+	case "monitor":
+		_, err = tx.Exec("GRANT SERVICE_CONNECTION_ADMIN ON *.* TO 'monitor'@'%'")
```

Comment:
please try to use syntax `/*!80015 <query> */;` like we already spoke in https://github.com/percona/percona-xtradb-cluster-operator/commit/8cb8b76c6

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/548/commits/811c5826cf3b04ce5aa0eda9c1621d7f47f93445